docs/modules/user-guide/partials/proc_migrating-commands.adoc

```diff
@@ -31,4 +31,4 @@ This section describes how to migrate plug-ins to devfile v2. There ARE major ch
 
 * Devfile metadata: add a link to an external website. See https://github.com/che-incubator/devworkspace-api/issues/38[corresponding issue].
 
-* Stack/Devfile Matching Rules. See https://github.com/che-incubator/devworkspace-api/issues/40[corresponding issue].
\ No newline at end of file
+* Stack/Devfile Matching Rules. See https://github.com/che-incubator/devworkspace-api/issues/40[corresponding issue].
```

Comment:
This Stack/Devfile Matching Rules  issue is still open and hasn't been implemented yet